modules/sdk-api/src/encrypt.ts

```diff
@@ -15,10 +15,11 @@ export function bytesToWord(bytes?: Uint8Array | number[]): number {
   return bytes.reduce((num, byte) => num * 0x100 + byte, 0);
 }
 
-export function encrypt(password: string, plaintext: string, {
-  salt = randomBytes(8),
-  iv = randomBytes(16),
-} = {}): string {
+export function encrypt(
+  password: string,
+  plaintext: string,
+  { salt = randomBytes(8), iv = randomBytes(16) } = {}
```

Comment:
Why do we include these params? New type?

Reply:
I'm not sure, but this isn't a change I made. I just `git mv` 'd this file. This is the same as it is currently:
https://github.com/BitGo/BitGoJS/blob/master/modules/bitgo/src/encrypt.ts#L21